pipelines/LSSTCam-imSim/ApVerify.yaml

```diff
@@ -15,6 +15,11 @@ tasks:
     class: lsst.ap.association.DiaPipelineTask
     config:
       doPackageAlerts: True
+  rbClassify:
+    class: lsst.meas.transiNet.TransiNetTask
+    config:
+      modelPackageName: 'neighbor:///rbResnet50-DC2'
```

Comment:
Note that, in order for this to work on Jenkins, we'd need to modify https://github.com/lsst-dm/jenkins-dm-jobs/blob/main/pipelines/scipipe/ap_verify.groovy to download the neighbor repository in addition to the `ap_verify` dataset.